src/views/RegistrationView.vue

```diff
@@ -30,78 +31,95 @@ const handleError = (error) => {
 	errorMessage.value = 'Failed to authenticate user'
 }
 
-const authenticateUser = async () => {
-	const requestBody = {
-		username: username.value,
-		password: password.value
-	}
-
-	const authHeaders = {
-		'Content-Type': 'application/json',
-		'Accept': 'application/json',
-	}
-
-	try {
-		await fetchApiCall({ url: store.state.observationPortalUrl + 'api-token-auth/', method: 'POST', body: requestBody, headers: authHeaders, successCallback: storeToken, failCallback: handleError })
-	} catch (error) {
-		handleError(error)
-	}
-}
-
 const storeUser = (user) => {
 	store.dispatch('userData/setUsername', username.value)
 	store.dispatch('userData/setUserProfile', user)
 	store.commit('setProjects', user.proposals)
-	router.push({ name: 'ProjectView' })
+	router.push({ name: 'DataSessions' })
```

Comment:
I think users should be rerouted to projects (since Wayne says that that's the first thing that users should see)

Reply:
Did this cause the project view doesn't have a button to go to data sessions so to get off the screen you have to create a session. But actually for now I can revert this and we can add a button to go to Data sessions on the project view in the case you don't want to make a new session.